apps/roam/src/utils/supabaseContext.ts

```diff
@@ -105,7 +107,7 @@ export const getSupabaseContext = async (): Promise<SupabaseContext | null> => {
         personName,
         personEmail,
       });
-      CONTEXT_CACHE = { platform: "Roam", spaceId, userId };
+      CONTEXT_CACHE = { api: window.roamAlphaAPI, platform: "Roam", spaceId, userId };
```

Comment:
This shouldn't be needed.  `roamAlphaAPI` is always available on the `window` object